src/sysadmin/audit.md

```diff
@@ -55,11 +67,77 @@ An audit type is an action that triggers an audit operation. Currently we suppor
+
+### trackedentityinstanceaudit
+
+| Column     | Type                        | Description |
+|------------|-----------------------------|-------------|
+| trackedentityinstanceauditid | integer | Primary key. |
+| trackedentityinstance | text  | Tracked entity instance name.  |
+| created  | timestamp without time zone | Time of creation. |
+| accessedby | text | Username of the user performing the audited operation. |
+| audittype | text | READ, CREATE, UPDATE, DELETE, SEARCH |
+| comment | text | The code of the audited object. |
+
+This data can be retrieved via [API](https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-master/audit.html#webapi_tracked_entity_instance_audits).
+
+### trackedentityattributevalueaudit
+
+| Column     | Type                        | Description |
+|------------|-----------------------------|-------------|
+| trackedentityattributevalueauditid | integer | Primary key. |
+| trackedentityinstanceid | integer | Instance ID of which the attribute value belongs to.  |
+| trackedentityattributeid | integer | Attribute ID.  |
+| created  | timestamp without time zone | Time of creation. |
+| modifiedby  | text | Username of the user performing the audited operation. |
+| audittype | text  | READ, CREATE, UPDATE, DELETE, SEARCH |
+| value | text | The value of the audited object. |
+| encryptedvalue | text | The encrypted value if confidentiality flag is set. |
+
+
+This data can be retrieved via [API](https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-master/audit.html#webapi_tracked_entity_attribute_value_audits).
+
+### trackedentitydatavalueaudit
+
+| Column     | Type                        | Description |
+|------------|-----------------------------|-------------|
+| trackedentitydatavalueauditid | integer | Primary key. |
+| programstageinstanceid | integer | Program stage ID of which the data value belongs to.  |
+| dataelementid | integer | ID of the data element.  |
+| created | timestamp without time zone | Time of creation. |
+| modifiedby | text | Username of the user performing the audited operation. |
+| audittype | text | READ, CREATE, UPDATE, DELETE, SEARCH |
+| value | text | The value of the audited object. |
+| providedelsewhere | bool | Indicates whether the user provided the value elsewhere or not. |
+
+This data can be retrieved via [API](https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-master/audit.html#webapi_tracked_entity_data_value_audits).
+
+## Breaking the glass
+Breaking the glass features allows to access records a DHIS2 user doesn't have access in special circumstances. As a result of such, users must enter a reason to access such records.
+
+A video explaining how it works can be found in our Youtube channel [here](https://www.youtube.com/watch?v=rTwg5Ix_E_M).
```

Comment:
I imagine this youtube video is much more helpful, but ideally our primary reference should be to documentation of that feature.

Reply:
Agree, but we don't have any. This is the only thing I was able to find: https://docs.dhis2.org/en/use/user-guides/dhis-core-version-236/tracking-individual-level-data/tracker-capture.html#break_glass

We do have some internal docs but not meant to be shared outside.